### PR TITLE
First relayers integration test from eth to elrond

### PR DESCRIPTION
**Pull request by iulianpascalau**
- first relayers integration test from eth to elrond

### CODE REVIEW

File: cmd/bridge/main.go [hunk 1]
```diff
@@ -204,3 +223,50 @@ func attachFileLogger(log logger.Logger, flagsConfig *relay.ContextFlagsConfig)
...
+	p2pConfig := config.P2PConfig{
+		Node:                nodeConfig,
+		KadDhtPeerDiscovery: peerDiscoveryConfig,
+		Sharding: config.ShardingConfig{
+			TargetPeerCount:         0,
+			MaxIntraShardValidators: 0,
+			MaxCrossShardValidators: 0,
+			MaxIntraShardObservers:  0,
+			MaxCrossShardObservers:  0,
+			Type:                    "NilListSharder",
```
Comment (bogdan-rosianu) [inline cmd/bridge/main.go:253]: _maybe_ extract const for these magic numbers/values

File: cmd/bridge/main.go [hunk 2]
```diff
@@ -204,3 +223,50 @@ func attachFileLogger(log logger.Logger, flagsConfig *relay.ContextFlagsConfig)
 
 	return fileLogging, nil
 }
+
+func buildNetMessenger(cfg relay.Config, marshalizer marshal.Marshalizer) (relayp2p.NetMessenger, error) {
```
Comment (bogdan-rosianu) [inline cmd/bridge/main.go:227]: could have been a relayer factory :smile:

Comment (iulianpascalau) [reply cmd/bridge/main.go:227]: soon :TM:
We will end-up creating factories that can be easily tested, of course.

File: cmd/bridge/main.go (hunk 1 above)
Comment (iulianpascalau) [reply cmd/bridge/main.go]: done

**Review by bogdan-rosianu — APPROVED** [review]

**Review by dragos-rebegea — APPROVED** [review]

**Pull request merged**